gcc/rust/checks/errors/borrowck/polonius/rust-polonius.h

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2020-2022 Free Software Foundation, Inc.
```

Comment:
ditto, plus 2022 seems dubious here, should be 2023.

Reply:
Yes, thanks.

Reply:
Fixed